src/diffpy/cmi/cli.py

```diff
@@ -72,6 +72,43 @@ def map_pack_to_examples() -> dict[str, List[str]]:
     return examples_by_pack
 
 
+def copy_examples(
+    examples: List[str],
```

Comment:
I though we were going to use a dict?  We may want to diagram this out a bit.  We could make this function lighter weight if we just hand it a list of paths and then copy everything in the path.  This is more general and reusable.  Then we need another function that builds that list of paths from our dict and so on.   Or we could do all that work of building the paths in here.  I am not sure which is better, but usually better separation is better, so the copy function just copies.  Then it would take a list of paths not a list of strings.